s29741-pj-JakubKolad/30.09/Z1.txt

```diff
@@ -0,0 +1,50 @@
+1.Uruchom terminal.ctrl+alt+t
+
+2.Wyświetl zawartość aktualnego katalogu.ls
+
+3.Sprawdź gdzie aktualnie się znajdujesz.pwd
+
+4.Przejdź do katalogu Pulpit za pomocą ścieżki względnej.cd Desktop/
+
+5.Sprawdź gdzie jesteś.pwd
+
+6.Wyświetl zawartość bieżącego katalogu.ls
+
+7.Wyświetl zawartość swojego katalogu domowego bez przechodzenia do niego (ls z odpowiednim parametrem).ls /home
+
+8.Przejdź do katalogu Obrazy (lub innego katalogu, który jest dostępny - podaj wprost, że zmieniasz katalog na inny bo np. Obrazy nie są u Ciebie dostępne) w twoim katalogu domowym za pomocą ścieżki względnej. cd Pictures/
+
+9.Sprawdź gdzie jesteś.pwd
+
+10.Zobacz do jakiego katalogu przejdziesz za pomocą ścieżki ././././././. Katalog nie uległ zmianie.
+
+*	W opisie rozwiązania proszę umieścić również to jak rozumiesz co się właściwie stało.	Dlaczego znalazła/eś się w tym miejscu do którego przeniosła Cię ta ścieżka?
+
+Rozumiem to jako polecenie "zostań tu gdzie jesteś". Po doczytaniu w Internecie rozumiem, że "./" pozwala np. na edytowanie plików "w dole" ścieżki,
+w której się obecnie znajdujemy. Innymi słowy jeżeli plik znajduje się w podkatalogach mojej obecnej lokalizacji to za pomocą "./" nie muszę ręcznie
+do tych katalogów przechodzić żeby go edytować.
+
+*	Czy dodanie kilku ./././././ na początku/środku/końcu powoduje, że ścieżka się zmieni?
+Nie.
+
+*	Co ta ścieżka oznacza/sobą reprezentuje?
+Umożliwia wykonywanie poleceń bez potrzeby zmiany bieżącego katalogu.
+
+11.Przejdź do swojego katalogu domowego za pomocą ścieżki bezwzględnej.
+cd /home
+
+12.Zobacz co się stanie jak wpiszesz komendę 'cd /root'. Jak już to zrobisz, zastanów się co się stało. Poszukaj w Internecie.
+
+Pojawił się komunikat "bash: cd: /root: Permission denied". Oznacza to, że mój użytkownik nie ma uprawnień do katalogu /root.
+Katalog ten znajduje się na szczycie drzewa folderów i jest katalogiem domowym superusera (administratora systemu). Jeżeli 
+znam hasło su albo należę do userów wymienionych w pliku sudoers, mogę uzyskać dostęp do katalogu /root.
+
+13.Zobacz, co się stanie, jeśli wciśniesz kombinację klawiszy: Ctrl+Alt+F3
+Sprawdź w Internecie co się dzieje kiedy używa się takiej kombinacji. W Internecie mogą pojawić się wersje z F1-F6. To de facto to samo.
+Odpowiedź "wylogowało mnie" jest błędna. Nie. Nie wylogowało. Szukaj dalej.
+
+Przełączyłem się z interfejsu graficznego na widok terminala, wszystkie polecenia mogę wykonywać jak wyżej. Za pomocą Ctrl+Alt+F7 mogę wrócić z powrotem do interfejsu graficznego.
+
+14.A teraz co się stanie po: Alt+F1 lub ALT+F2 lub ALT+F7 (na starszych linuxach)
+
+Przełączam się między terminalami, w każdym z nich poza 1 mogę pracować.
```

Comment:
Dobrze ale chodziło o powrót do interfejsu graficznego.
Z uwagi na to, że kombinacja klawiszy uległa zmianie na przestrzeni lat, to instrukcja musi brać to pod uwagę. Więc stąd 3 kombinacje. W twoim przypadku chodziło o `Alt+F7`.